charts/datadog/templates/_container-security-agent.yaml

```diff
@@ -107,6 +107,16 @@
       subPath: system-probe.yaml
     {{- end }}
     {{- end }}
+    {{- if .Values.datadog.kubelet.hostCAPath }}
```

Comment:
I'm not sure that the security-agent need to have access to the kubelet. 
Do we have a compliance check that run against the kubelet api?

Reply:
I guess we don't need it with remote tagger

Reply:
we might need it later if we start to use the kubelet `/pods` list for compliance check. but we can add it later